DevCycle/DVCVariable.swift

```diff
@@ -61,6 +63,20 @@ public class DVCVariable<T> {
         }
     }
     
+    @objc func propertyChange(notification: Notification) {
+        guard let userConfig = notification.userInfo?["new-user-config"] as? UserConfig else {
+            return
+        }
+        if let variableFromApi = userConfig.variables[key] {
+            self.update(from: variableFromApi)
+        } else if (!isEqual(self.value, self.defaultValue)) {
+            self.value = self.defaultValue
```

Comment:
maybe this block could go inside a `resetToDefault` method?

also is there a case here where the variable is no longer in the config, but its previous value was the same as the default value? In that case you wouldn't want to call the handler, but you would want to set `isDefaulted` back to true